workspaces/mm-container-build/config.xml

```diff
@@ -62,7 +62,7 @@ mv mattermost/* app/app_code/</command>
       <dockerfilePath>app/Dockerfile</dockerfilePath>
       <skipBuild>false</skipBuild>
       <skipDecorate>false</skipDecorate>
-      <repoTag>${BUILD_NUMBER}</repoTag>
+      <repoTag>${GIT_COMMIT}</repoTag>
```

Comment:
+1 on this identifier moving through. I like the idea of having the git commit be the unique identifier, until we get into having multiple revisions of that single git commit in all our ci infra, and trying to decouple what happened where. I think there's room for improvement here but I don't have a pliable solution today.

Perhaps this works fine as is and I'm over-thinking it too :)